_docs/geyser/geyser-command-line-arguments-and-system-properties.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Geyser Command Line Arguments and System Properties
+---
+
+# Geyser Command Line Arguments and System Properties
+Geyser offers a few command line arguments/system properties to allow you to configure Geyser without editing the config files.
+Additionally, you can suppress some warnings that may be printed to the console.
+
+### Configuration system properties
+You can set Geyser to automatically bind to a specific address and port by using the following command line arguments. <br>
+This is primarily aimed at server hosting providers to automatically configure servers for users.
+
+<div class="alert alert-info" role="alert">
+	Note: the Geyser specific properties are prioritized over the plugin properties!
+</div>
+
+- ```-DgeyserUdpPort=server``` or ```-DpluginUdpPort=server```
+  - ```-1``` means UDP is not supported and will forcibly stop Geyser.
+  - ```server``` means to match the port of the TCP server.
+  - any other number means to use that specific port
+
+- ```-DgeyserUdpAddress=server``` or ```-DpluginUdpAddress=server```
+  - ```server``` means to match the bind address of the TCP server
+  - any other string will be used as-is for the bind address.
+
+### Disabling warnings and advanced configuration
+You may disable some warnings that may be printed to the console by using the following command line arguments:
+
+<div class="alert alert-danger" role="alert">
+	Warning: Disabling Geyser warnings from being logged will not fix the real issue! Only disable them if you know what you are doing. 
+</div>
+
+- ```-DGeyser.PrintSecureChatInformation=true```
+  - Allows you to disable the warning about secure chat being disabled.
```

Comment:
I'd note this doesn't do much anymore because the warning is now only printed if the server sends the "chat message wasn't sent because missing profile key" message.